regulations/check.py

```diff
@@ -0,0 +1,122 @@
+import sys
+
+
+class translationChecker():
+
+  def __init__(self):
+    sys.stdout.flush()
+    self.regulations_set = self.readFile('wca-regulations/wca-regulations.md', False)
+    self.guidelines_set = self.readFile('wca-regulations/wca-guidelines.md', True)
+
+  # If isGuidelines is False, check that every regulation is indented
+  # according to its number, that every lind number doesn't end with '+'.
+  #
+  # If isGuidelines if True, check that every regulation has no
+  # indentation, that every line number ends with '+', that every
+  # line has a label, and there are six different labels.
+  #
+  # Returns set of regulations numbers.
+  def readFile(self, filename, isGuidelines):
+    numbers_set = set()
+    found_table_of_contents = False
+    labels_set = set()
+
+    with open(filename, 'r') as input_file:
+      for line_number, line in enumerate(input_file):
+        line_number += 1
+        line = line.rstrip('\n')
+
+        # Skip everything before <table-of-contents>.
+        if not found_table_of_contents:
+          if line == '<table-of-contents>':
+            found_table_of_contents = True
+          continue
+
+        # Ignore empty lines or lines that start with '<' or '#'.
+        if line == '' or line[0] == '<' or line[0] == '#':
+          continue
+
+        # Try to extract regulation number. If there is no number, skip.
+        ind1, ind2 = line.find('- '), line.find(')')
+        if ind1 == -1 or ind2 == -1:
+          continue
+        line_id = line[ind1 + 2 : ind2]
+        numbers_set.add(line_id)
+
+        if isGuidelines:
+          # Check the absence of indentation.
+          if line[0] != '-':
+            print "Warning: %s:%d: line has indentation, while it shouldn't." % (filename, line_number)
+            print line
+            print
+
+          # Check that there is a label.
+          ind1, ind2 = line.find('['), line.find(']')
+          if ind1 == -1 or ind2 == -1:
+            print "Warning: %s:%d: line has no label in brackets." % (filename, line_number)
+            print line
```

Comment:
Ok, I'll add it. Though in any case this would trigger another warning that you have 7 different labels instead of 6 (one of them is empty string), but this would be more transparent.